test/blackbox-tests/test-cases/melange/promote.t

```diff
@@ -33,3 +33,40 @@ Targets are promoted to the source tree
 
   $ dune clean
   $ ls ./dist
+
+Promotion with targets in a directory
+
+  $ rm -rf ./dist ./hello.ml ./dune
```

Comment:
How about just moving this test into its own file and removing this clean up? It would make things less fragile and execute things in parallel when possible to boot.

Reply:
absolutely, doing that now